README.md

```diff
@@ -3,35 +3,82 @@
 ## Overview
 The edX learning platform's frontend for enterprise learners.
 
-## Getting started
+## Setup for development
 
-- Clone this repository locally with `git clone <repo URL>`
+Install nodejs (using nvm maybe a good idea to be able to switch node versions and because there is an .nvmrc file that will be usable by nvm)
 
-##### How to set up the dev environment
 To run the project, first install dependencies
 
-`$ npm i`
+`$ nvm use` # will install/setup node needed for this project
+`$ npm i`   # will install deps
 
 then, start the dev server:
 
 `$ npm start`
 
-### Where is it running?
-The project itself will run on http://localhost:8734
+Visit http://localhost:8734
```

Comment:
Should this mention that you need to be at http://localhost:8734/{enterprise-slug}? The root http://localhost:8734/ currently just renders a 404.